src/Features/CSharp/Portable/Completion/CompletionProviders/PropertySubPatternCompletionProvider.cs

```diff
@@ -124,7 +124,7 @@ private static SyntaxToken TryGetOpenBraceOrCommaInPropertyPatternClause(SyntaxT
                 return default;
             }
 
-            return token.Parent.IsKind(SyntaxKind.PropertyPatternClause) ? token : default;
+            return token.Parent.IsKind(SyntaxKind.PropertyPatternClause, SyntaxKind.ListPatternClause) ? token : default;
```

Comment:
Is there a test for this change?

Reply:
Lots of failing tests for this completion provider.

`{ $$ }` always see a list pattern clause.